dstopic/packages/dstopic-command/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub trait Command {
+    fn execute(self) -> Result<(), Box<dyn std::error::Error + Send + Sync>>;
```

Comment:
Would we ever want to be able to return an exit code directly instead of `()` (for example, from a test runner?)

Reply:
To me, non-zero exit codes are always errors, so I think this belongs in the domain of the Error half of Result. I don't really see a particular reason right now to allow returning these codes, and I think if we have some super-specialized case, there's nothing stopping folks from just doing `std::process::exit()` themselves, since the stack at the Command level is super-shallow. I'll keep thinking about this and see if there's a better way going forward, though -- I'd just rather not do exit codes through the value side.